ci/releases.yml

```diff
@@ -61,3 +61,17 @@ release commit messages:
       - /^release\//
   script:
     - ci/scripts/check_release_commit_messages.sh
+
+# Messaging system deploy to production
+msg-system codesign deploy:
+  stage: deploy to production
+  when: manual
+  needs:
+    - "msg-system config sign stable"
+  only:
+    refs:
+      - codesign
+  tags:
+    - darwin
```

Comment:
Let's change it to deploy runner and we're good to go.

Reply:
Fixed.